### ISSUE TITLE
Feature Request: Ability to disable Key Backup Prompt

### ISSUE THREAD

**Issue by jkms** [vector-im/riot-web#9256]
**Is your suggestion related to a problem? Please describe.**
I'd like to be able to disable the "Key Backup" prompt for users

![image](https://user-images.githubusercontent.com/660916/54836268-1a76db80-4c81-11e9-9543-5624f81b8250.png)

**Describe the solution you'd like**
 A Parameter in the config.json to toggle key backup

**Describe alternatives you've considered**
The only other option I have is training my users

**Additional context**
I have disabled room encryption on my homesever as per #8067.  I have a legal need to require the contents of my homeserver are auditble, and this is working well.  The prompt about encrypted messages is confusing my users, and disabling it would make things easier.

**Comment by dandv** [comment]
Second that. My casual user friends who don't want the key backup have been very annoyed by the prompts.

**Comment by emorrp1** [comment]
I would also like to disable the prompt for users on my homeserver, but only because I don't think it's necessary any more with cross-signing unless you lose every single device?

As a user I'm least likely to want key backup on matrix.org because then e2ee is only as good as the passphrase, but more likely to want key backup on a trusted friend's homeserver.

**Comment by Bun-Bun** [comment]
This could tie into a Synapse config option, if e2ee is disabled disable all related prompts. 

https://github.com/matrix-org/matrix-doc/issues/2528

**Comment by pdurbin** [comment]
Hi! I assume this is the issue to follow if I'm anticipated that my users will be frustrated with the inability to turn off the "Set up Secure Backup" prompt in the Element desktop client, as shown in the screenshot below. ("Safeguard against losing access to encrypted messages & data".) If I should create a new issue instead, please just let me know and I'll be happy to. I'm coming from IRC and overall I'm having a good experience with Matrix. Thanks for maintaining a nice client!

<img width="426" alt="Screen Shot 2021-06-03 at 10 28 36 AM" src="https://user-images.githubusercontent.com/21006/120662243-f913fb80-c456-11eb-94f8-709344f2587d.png">

**Comment by yennor** [comment]
Yes, it would be really nice to be able to get rid of that messge, it annoys me each time i restart the computer. Especially
I already got setup a proper backup system for all my data, so I don't need to separately stores those information.

**Comment by HugoZink** [comment]
2 years later, this message is still incredibly annoying. I think I might have to switch clients.

**Comment by SamuelAllain** [comment]
That is indeed very annoying

**Comment by raphaelstary** [comment]
<img width="298" alt="image" src="https://user-images.githubusercontent.com/907724/236409488-5ac70aa9-71a1-441a-9546-364ae7234a74.png">

I use multiple desktop PCs during the day and therefore I get to enjoy this prompt multiple times a day.

At first I thought, it's just a tiny dialog - no biggie - just move on ... but lately it really pisses me off :(
